org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Positioner.java

```diff
@@ -92,20 +92,20 @@ public IPosition getPosition() throws ScanningException {
 	}
 
 	@Override
-	public List<IScannable<?>> getMonitors() throws ScanningException {
+	public List<IScannable<?>> getMonitorsPerPoint() throws ScanningException {
 		// TODO Use the _Scannable which is a remote scannable connection.
 		return null;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)